src/marqo/tensor_search/add_docs.py

```diff
@@ -61,7 +64,7 @@ def threaded_download_and_preprocess_images(allocated_docs: List[dict], image_re
             for field in list(doc):
                 if field not in tensor_fields:
                     continue
-                if isinstance(doc[field], str) and clip_utils._is_image(doc[field]):
+                if force_download or (isinstance(doc[field], str) and clip_utils._is_image(doc[field])):
```

Comment:
String check seems redundant as it's covered by `is_image`

Reply:
Agreed